src/lib/styles/global/colors.scss

```diff
@@ -121,6 +121,7 @@
   --violet-650: #150c31;
   --violet-900: #130a2c;
 
+  --night-50: rgb(155, 173, 209);
```

Comment:
Why is this `rgb` while all the others are `#XXXXXX`?

Reply:
Good catch, this should be hex.